src/app.js

```diff
@@ -38,13 +38,19 @@ if (process.env.KEEP_ALIVE) {
 	});
 }
 
+const BODYPARSER_JSON_LIMIT = process.env.BODYPARSER_JSON_LIMIT || '100kb';
+if (process.env.BODYPARSER_JSON_LIMIT === undefined) {
+	/* eslint-disable-next-line  */
+	logger.warning(`please set the environment variable BODYPARSER_JSON_LIMIT to 12mb for helpdesk to work correctly! (Currently: ${BODYPARSER_JSON_LIMIT})`);
```

Comment:
this warning then can be moved to the controller but should be printed on application start...